randovania/games/samus_returns/logic_database/Area 2 - Dam Exterior.txt

```diff
@@ -1082,8 +1083,10 @@ Extra - asset_id: collision_camera_040
                   All of the following:
                       Lay Bomb
                       Spring Ball or Mid-Air Morph (Beginner)
```

Comment:
Like above, the pretty print is fine, but the DB looks off. Make it cleaner
![image](https://github.com/randovania/randovania/assets/38679103/715cfe72-dc94-47c1-9261-87004eab5151)